html-templates/simple.mustache

```diff
@@ -294,11 +294,12 @@
 			});
 		});
 	</script>
+	<title>Salesforce Code Analyzer Report</title>
 </head>
 
 <body>
     <h1 id="reportTitle">Salesforce Code Analyzer Report</h1>
-	<div id="summaryChart"/></div>
+	<div id="summaryChart"></div>
```

Comment:
Fixes w3 validation error:
`Self-closing syntax (/>) used on a non-void HTML element. Ignoring the slash and treating as a start tag.`